README.md

```diff
@@ -39,7 +39,7 @@ It's not possible to add dependencies between monorepo packages using yarn (ie:
 Unpublished dependencies between monorepo packages can be linked by:
 
 - Manually adding the expected dependency to `package.json` (ie: `@verida/encryption-utils`)
-- Run `npx lerna bootstrap` in the root directory of this project
+- Run `yarn install` in the root directory of this project
```

Comment:
@nick-verida This change is incorrect, please revert.

Reply:
@tahpot yes, `lerna bootstrap` has been replaced:

> Replace lerna bootstrap with npm install (or yarn/pnpm). If you are already performing your package manager's install command somewhere in your workflow before where you had previously called lerna bootstrap, then you can just delete it instead. 

https://lerna.js.org/docs/legacy-package-management